### DISCUSSION TITLE
Setting to open a post directly when clicking on its card

### DISCUSSION THREAD

**Discussion by lukefiji** [dailydotdev/daily#874] [Feature Requests]
Just wanted to suggest adding a setting to open an a post directly when clicking on its card, instead of opening its discussion modal.

PS - love what your team is doing - it was no-brainer for me to switch on over from Muz.li the moment I learned about it :)

**Comment by idoshamun** [comment]
Hi @lukefiji,

First of all, thanks for all the kind words. 🤩
At the moment, there is no setting to achieve that but you can click the "read post" button to go directly to the article.
<img width="360" alt="image" src="https://user-images.githubusercontent.com/1993245/233269693-6a7aae8e-876c-4d5a-8d3b-044626a42f59.png">
